lib/src/components/NxCollapsibleItemsSelect/NxCollapsibleRadioSelect/stateful/__tests__/NxStatefulCollapsibleRadioSelect.test.tsx

```diff
@@ -429,6 +430,28 @@ describe('NxStatefulCollapsibleRadioSelect', function() {
       expect(keyEvent.defaultPrevented).toBe(false);
     });
 
+    it('does not submit the form when the clear filter button is clicked', async function() {
+      const user = userEvent.setup(),
+          onSubmit = jest.fn(),
+          view = render(
+            <NxForm onSubmit={onSubmit} showValidationErrors={false} >
+              <NxStatefulCollapsibleRadioSelect {...minimalProps} filterThreshold={2} />
+            </NxForm>
+          ),
+          inputEl = view.getByRole('textbox');
+
+      await runTimers();
+      inputEl.focus();
+      await user.keyboard('b');
```

Comment:
> Can't we just use `await user.type(inputEl, 'b');`?

For this (and the comment below), I was trying to match whatever code was there already (probably unnecessary!)- so NxCollapsibleRadioSelect used `user.keyboard('b')`, and NxSearchDropdown used regex instead of the string. I can change both if you prefer!

Reply:
> > Can't we just use `await user.type(inputEl, 'b');`?
> 
> For this (and the comment below), I was trying to match whatever code was there already (probably unnecessary!)- so NxCollapsibleRadioSelect used `user.keyboard('b')`, and NxSearchDropdown used regex instead of the string. I can change both if you prefer!

> I was trying to match whatever code was there already (probably unnecessary!)- so NxCollapsibleRadioSelect used `user.keyboard('b')`

Let's change it to `user.type` just to be consistent with the other tests.

For string vs regex: Again, just for the sake of consistency, let's stick with one. I'll leave it up to you to choose whether it be regex or string 😄

Reply:
> Let's change it to `user.type` just to be consistent with the other tests.

Sounds good! Done here: [1587b8c](https://github.com/sonatype/sonatype-react-shared-components/pull/956/commits/1587b8c4de7a53b84fb811f7752135222d569b43)